base/strings/util.jl

```diff
@@ -464,6 +464,55 @@ function hex2bytes(s::AbstractString)
 end
 
 """
+    hex2bytes(d::Vector{UInt8}, s::Vector{UInt8}, nInBytes::Int=length(s))
+
+Convert the first `nInBytes` hexadecimal bytes array to its binary representation. The
+results are populated into a destination array. The function returns the number of bytes
+copied into the destination array. The size of destination array must be at least half of
+the `nInBytes` parameter.
+"""
```

Comment:
`hex2bytes` now has an example. 

Typically, when we have both an in-place and out-of-place version of a function, we only give examples in the simpler out-of-place version.